core/dbt/contracts/graph/parsed.py

```diff
@@ -592,7 +592,8 @@ class ParsedSourceDefinition(
     UnparsedBaseNode,
     HasUniqueID,
     HasRelationMetadata,
-    HasFqn
+    HasFqn,
```

Comment:
I'm not a Python person, but this looks weird to me to add the comma and the newline. Is that a formatting thing?

Reply:
Yeah-- I'm starting to use an auto-formatter called Black which is opinionated.  Not sure how much I like it though.
:)